step.yml

```diff
@@ -273,13 +305,60 @@ inputs:
     - "no"
     is_required: true
 
+- certificate_url_list: $BITRISE_CERTIFICATE_URL
```

Comment:
These are not debugging related inputs, all automatic code signing related inputs should be under `Automatic code signing` category.